user_docs/en/userGuide.t2t

```diff
@@ -1220,8 +1220,9 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
-==== Report line indents with ====
-This option allows you to configure how indentation at the beginning of lines is reported. The Report line indents with combo box has four options.
+==== Report line indentation with ====
+This option allows you to configure how indentation at the beginning of lines is reported.
+The Report line indents with combo box has four options.

```

Comment:
You missed another instance of "Report line indents with" -> "Report line indentation with". :)